packages/button/src/styles/default.module.css

```diff
@@ -25,6 +25,7 @@
   line-height: 1.2;
   border-style: solid;
   white-space: nowrap;
+  gap: var(--gap-xs);
```

Comment:
can we use the spacing variables instead of gap from now on
packages/vars/src/spacing.css